src/core/lib/iomgr/exec_ctx.cc

```diff
@@ -51,14 +47,8 @@ static void exec_ctx_run(grpc_closure* closure, grpc_error_handle error) {
 }
 
 static void exec_ctx_sched(grpc_closure* closure, grpc_error_handle error) {
-#if defined(GRPC_USE_EVENT_ENGINE) && \
```

Comment:
Leaving a comment here that I'm fine to remove this, I don't think it'll help us with our current plans to integrate EventEngine::Run and RunAt.

Reply:
Oh totally intended to call this out .. yeah that was my assumption too.